panels/move.py

```diff
@@ -13,6 +14,15 @@ class Panel(ScreenPanel):
 
     def __init__(self, screen, title):
         super().__init__(screen, title)
+
+        if self.ks_printer_cfg is not None:
+            dis = self.ks_printer_cfg.get("move_distances", '0.1, 0.5, 1, 50, 10, 25, 50')
+            if re.match(r'^[0-9,\.\s]+$', dis):
+                dis = [str(i.strip()) for i in dis.split(',')]
+                if len(dis) <= 7:
```

Comment:
if 1 < len(dis) <= 7:

Reply:
the list should have at least 2 items

Reply:
> if 1 < len(dis) <= 7:

I tried initially but it didn't work. I'll try again..

> the list should have at least 2 items

ok